.gitignore

```diff
@@ -3,3 +3,12 @@ gen
 build
 .settings
 source/res/values/com_crashlytics_export_strings.xml
+.idea/
+*.iml
+*.iws
```

Comment:
See [here](https://github.com/joindin/Joind.in-android-app/pull/22#discussion_r33491581) - these should be handled on your local development environment really.